### PR TITLE
Added styles in regard of imports sentences

### PR DESCRIPTION
**Pull request by m-revetria**
Additionally:
* Fixed some minor issues in the code style of the provided examples
* Fixed updated some old code to with Swift 3 syntax
* Added SwiftLint configuration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,6 +201,58 @@ let numberOfWheels = 4
...
+Avoid duplicated imports.
+
+**prefered**
+```swift
+//
+//  AppDelegate.swift
+//  TestingHero
+//
+//  Created by ?? ??? on 1/1/17.
+//  Copyright © 2017 Xmartlabs. All rights reserved.
```
Comment (enrigalmig) [inline README.md]: maybe the comment part is not required to show the best way to do the imports, what do you think?

File: README.md [hunk 2]
```diff
@@ -200,6 +201,58 @@ let numberOfWheels = 4
...
 
+### Imports
+
+Refer to [Import Declarations from Apple Doc].
+
+Imports should be declared at top of each Swift file and alphabetically (ignoraing case) sorted. There could be any number of comments before the first import sentence.
+
+Keep imports that declare a `kind` first ordered by their `kind` after imports that don't do it. Optionally group imports by kind declaration into logical blocks.
+
+Imports marked with `@testable` should be put at the end of the list of imports. Optionally you a trailing line may be put in order to group normal imports and `@testable` imports in two different logical blocks.
```
Comment (santiagofm) [inline README.md]: Optionally you may add a line in order to group..

**Review by enrigalmig — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mats-claassen) [inline README.md]: `Optionally an empty line`

File: README.md (hunk 1 above)
Comment (m-revetria) [reply README.md]: Yup, 'm going to simplify it. I want to let clear that comments are allowed before imports

**Pull request merged**